librus/__init__.py

```diff
@@ -196,13 +199,11 @@ def __init__(self, date, lesson, teacher, category, subject,
 
 
 class Grade(object):
-    def __init__(self, grade, comment, title, added_date, teacher, correction_grade, added_by):
+    def __init__(self, lesson_name, grade, date, teacher, added_by):
+        self.lesson_name = less
         self.grade = grade
-        self.comment = comment
-        self.title = title
```

Comment:
Why remove?

Reply:
This method doesn't get the comment, and category is the same as title.